frontend/src/components/BarChartWidget.tsx

```diff
@@ -152,7 +152,11 @@ const BarChartWidget = (props: Props) => {
       )}
       {data && data.length && (
         <div aria-hidden="true">
-          <ResponsiveContainer width="100%" height={dims.chartHeight}>
+          <ResponsiveContainer
+            width="100%"
+            height={dims.chartHeight}
+            id={`${props.title}-bar-chart`}
```

Comment:
How we ensure that `props.title` meets the html id attribute format criteria?
>The id attribute must contain at least one character, cannot start with a number, and must not contain whitespaces (spaces, tabs, etc.).

Please, review it as well in the below `chart` components.

Reply:
I think we should go with the id instead.

Reply:
I updated the ID.